tests/test.rs

```diff
@@ -26,6 +27,7 @@ mod tests {
         assert_eq!(bytes.len(), measure(&record));
 
         // decode from binary data
+        let mut bytes = AlignedBytes::<Struct>::new(&mut bytes);
```

Comment:
`decode()` now expects its input data to be properly aligned, and this is a minimally invasive way to prepare said data.